### PR TITLE
Prevent touch events from scrolling the page when dragging the page vertically in mobile devices

### PR DESCRIPTION
**Pull request by mandx**
Some users are reporting that Builder content using our Carousel widget can't be scrolled up and down when the touch event starts on a carousel slide.

This is an attempt to fix it by using CSS's `pointer-events` on slide images.

### CODE REVIEW

**Review by mrkoreye — APPROVED** [review]

File: packages/widgets/src/components/Carousel.tsx [hunk 1]
```diff
@@ -29,26 +29,24 @@ interface CarouselProps {
 
 // TODO: change to slick grid
 export class CarouselComponent extends React.Component<CarouselProps> {
-  divRef: HTMLElement | null = null;
-  sliderRef: Slider | null = null;
+  divRef = React.createRef<HTMLDivElement>();
```
Comment (mrkoreye) [inline packages/widgets/src/components/Carousel.tsx:32]: Is this part of the fix or just some refactoring?

Comment (mandx) [reply packages/widgets/src/components/Carousel.tsx:32]: Just refactoring

**Comment by steve8708** [comment]
merge?

**Comment by mandx** [comment]
@steve8708 should I also make another PR for the `webcomponents` code to reference the new version if this `widgets` package?

**Comment by steve8708** [comment]
ya please do

**Pull request merged**